tools/myexample2/seqtk_seq.xml

```diff
@@ -0,0 +1,96 @@
+<tool id="seqtk_seq" name="Convert to FASTA" version="0.1.0" python_template_version="3.5">
+    <requirements>
+        <requirement type="package" version="1.2">seqtk</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+	    seqtk seq 
+	    $shift_quality 
+	    -q $quality_min 
+	    -X $quality_max 
+	    #if $mask_regions 
+	    	-M '$mask_regions' 
+	   #end if
+	   #if $sample.sample
+	   	-f $sample.fraction 
+	   	-s $sample.seed
+	   #end if
+	   -a '$input1'> '$output1'
+    ]]></command>
+    <inputs>
+	    <param type="data" name="input1" format="fastq" />
```

Comment:
Please use 4 spaces for all indenting, it seems variable across the file

Reply:
Accomplished.